onnxruntime/core/graph/graph_utils.cc

```diff
@@ -202,40 +202,42 @@ static int GetIndexFromName(const Node& node, const std::string& name, bool is_i
   return static_cast<int>(index);
 }
 
-#endif  // !defined(ORT_MINIMAL_BUILD) || defined(ORT_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
-
-//----------------------------
-//--- end of local helpers ---
-//----------------------------
-
-#if !defined(ORT_MINIMAL_BUILD)
+bool MatchesOpSinceVersion(const Node& node, std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion> versions) {
```

Comment:
so these functions are not included in the header file in a minimal build? seems trickier to have them conditionally be "local helpers"

Reply:
if IsSupportedOptypeVersionAndDomain() is moved outside of `#if !defined(ORT_MINIMAL_BUILD)` in the header, should the functions that it depends on be moved also?

Reply:
I can move them. I only did it in the .cc as it seemed cleaner as I only need one of the two MatchesOpSinceVersion variants. If someone needs to directly use those methods in the future they should quickly discover they just need to update the header.